_data/companies.yml

```
@@ -111,19 +111,19 @@
   last_update: 2020-03-09
 
 - name: Avid
-  wfh: Encouraged
-  travel: Restricted
-  visitors: Restricted
-  events: Restricted
-  last_update: 2020-03-12
-
-- name: Bitwala
-  wfh: Encouraged
-  travel: Restricted
-  visitors: "?"
-  events: Restricted
-  last_update: 2020-03-12
-
+  wfh: Encouraged 
+  travel: Restricted 
+  visitors: Restricted 
+  events: Restricted 
+  last_update: 2020-03-12  
+
+- name: Bitwala 
+  wfh: Encouraged 
+  travel: Restricted 
+  visitors: "?" 
+  events: Restricted 
+  last_update: 2020-03-12 
+  
```

```suggestion
  wfh: Encouraged
  travel: Restricted
  visitors: Restricted
  events: Restricted
  last_update: 2020-03-12

- name: Bitwala
  wfh: Encouraged
  travel: Restricted
  visitors: "?"
  events: Restricted
  last_update: 2020-03-12

```

Stripping trailing whitespace